setup.py

```diff
@@ -22,7 +22,7 @@
     long_description = fh.read()
 
 requirements = [
-    "qiskit>=0.32",
+    "qiskit>=0.32,<=0.42.1",
```

Comment:
```suggestion
    "qiskit>=0.42.1",
```

Should it not be that?

Reply:
It's to prevent this issue in a test:
> qiskit.exceptions.QiskitError: "Error decomposing node of instruction 'rx': 'NoneType' object has no attribute 'global_phase'. Unable to define instruction 'u' in the basis."

Though @timmysilv, have you tried fixing the issue?

Reply:
i have not, but I figured this week isn't the best time to do it. i'll give it a shot today

Reply:
turns out Qiskit/qiskit-terra#9343 changed the behaviour of the "unroller" transpilation method option to account for target ops, and this broke RX badly. the intention of the test is only to show that we can pass custom transpilation options, so I've changed it to just use the "translator" option! those should pass with the latest qiskit (confirmed locally)